src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -3477,28 +3507,27 @@ private bool TryNextResult(out bool more)
 
         /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlDataReader.xml' path='docs/members[@name="SqlDataReader"]/Read/*' />
         // user must call Read() to position on the first row
-        override public bool Read()
+        public override bool Read()
         {
             if (_currentTask != null)
             {
                 throw SQL.PendingBeginXXXExists();
             }
 
             bool more;
-            bool result;
 
             Debug.Assert(_stateObj == null || _stateObj._syncOverAsync, "Should not attempt pends in a synchronous call");
-            result = TryReadInternal(true, out more);
+            TdsOperationStatus  result = TryReadInternal(true, out more);
```

Comment:
nit: remove extra space

Reply:
I did say "I know there's still some cleanup to do so no need to nitpick style/naming." 
It'll get cleaned up if/when I feel it's ready for wider review. At the moment I just want to try and get the CI to run over it.

Reply:
Done.